apis/kueue/v1alpha1/resourceflavor_webhook.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/klog/v2"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+// log is for logging in this package.
+var resourceFlavorLog = ctrl.Log.WithName("resource-flavor-webhook")
+
+func (rf *ResourceFlavor) SetupWebhookWithManager(mgr ctrl.Manager) error {
+	return ctrl.NewWebhookManagedBy(mgr).
+		For(rf).
+		Complete()
+}
+
```

Comment:
can you also add validation on the labels please? you can call this function https://github.com/kubernetes/kubernetes/blob/894a8fd4dd7fe40be5c035c03ce40c088c775521/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation.go#L71

Reply:
done